themes/jb/assets/js/jb-live.js

```diff
@@ -63,4 +63,26 @@ async function jbLive() {
     const [liveShow, archivedShow] = await Promise.allSettled([liveShowQuery, archivedShowQuery])
 
     return getEmbedLink(liveShow)?.toString() ?? getEmbedLink(archivedShow)?.toString();
-}
\ No newline at end of file
+}
+**
```

Comment:
missed a `/` for the start of your comment
```suggestion
/*
```